bounceview-ohos/src/main/java/com/github/hariprasanths/bounceview/CustomPopupDialog.java

```diff
@@ -47,36 +62,48 @@ public Component getCustomComponent() {
      * @return component set by setText, setBackColor
      */
     public Component getBaseComponent() {
-        return mTempComponent;
+        return mBaseComponent;
     }
 
     @Override
     public CustomPopupDialog setCustomComponent(Component component) {
         super.setCustomComponent(component);
-        this.mComponent = component;
+        this.mCustomComponent = component;
         return this;
     }
 
+    /**
+     * text will be set only to the baseLayout(not to to customLayout )similar to platform's PopupDialog behaviour.
+     *
+     * @param text to set
+     * @return base component where text field set to given parameter
+     */
     @Override
     public PopupDialog setText(String text) {
-        Text contentText = (Text) customLayout.findComponentById(ResourceTable.Id_temp_text);
+        Text contentText = (Text) basePopupLayout.findComponentById(ResourceTable.Id_temp_text);
```

Comment:
please rename the temp_text to base_text

Reply:
done